pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -337,6 +339,23 @@ func Newk8sOrchestrator(ctx context.Context, controllerClusterFlavor cnstypes.Cn
 	return k8sOrchestratorInstance, nil
 }
 
+func getReleasedVanillaFSS() map[string]string {
+	return map[string]string{
```

Comment:
nit: As you are not really using the value, you can instead use `map[string]struct{}`.

Reply:
Thanks! I've made the change.